.github/workflows/branch.yml

```diff
@@ -1,14 +1,7 @@
 name: Build non-master/main branches. And deploy specific ones only.
```

Comment:
Bør sikkert endre name og fjerne kommentarene og så legge på en fersk kommentar. Hva gjør denne? Får man en ny action i github man kan trykke på hvis en vil deploye?

Jeg ønsker fremdeles å bygge branchene mine, men det er kjekt og ha en action i github for å trigge deploy.

Reply:
Kommentarer osv bør nok oppdateres ja. Dette skal gjøre at det er litt enklere å deploye ikke-master-brancer; se https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/ for mer info. Det vil også gjøre det enklere å faktisk deploye master manuelt, da man ikke lenger vil være nødt til å lete seg fram til siste masterbygg for så å trigge re-run på det.

Reply:
Burde vi isåfall ha to? En som trigges på branch-push og bare bygger ikke-master-branches, og en som trigges på `workflow_dispatch` som bare gjør deploy?

Reply:
Jeg stemmer for det. Vi har en prod_deploy.yml. Kanskje branch_deploy.yml

Reply:
Enig. Jeg glemte at denne også var ansvarlig for å bygge branchene 😅